Equipment.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Equipment.h"
+#include "Laundromat.h"
+#include "WashingMachine.h"
+#include "Restaurant.h"
+#include "Server.h"
+#include "Speaker.h"
```

Comment:
If you're already including all of these in the header file, you don't need to include them in this cpp file
- [ ] revise